app/models/user.rb

```diff
@@ -38,9 +38,17 @@ class User < ApplicationRecord
   validates :email, uniqueness: true
 
   has_many :comments, foreign_key: :author_id
-  has_many :conversations
   has_many :company_conversations, foreign_key: :company_representative_id, class_name: 'Conversation'
   has_many :customer_conversations, foreign_key: :customer_id, class_name: 'Conversation'
   has_many :grader_ratings, foreign_key: :grader_id, class_name: 'Rating'
   has_many :gradee_ratings, foreign_key: :gradee_id, class_name: 'Rating'
+
+  def calculate_average_rating
+    sum = 0
+    self.grader_ratings.each do |a_grade|
+      sum = sum + a_grade.grade
+    end
+    return 1.0 * sum / (0.5*self.ratings_count)
```

Comment:
Ruby method automatically return the last line of the method, so you can omit the `return` here.

```suggestion
    1.0 * sum / (0.5*self.ratings_count)
```

Reply:
Done thanks